templates/SimpleInventory/edit-board.html

```diff
@@ -0,0 +1,45 @@
+{% extends "SimpleBase/base.html" %}
+
+{% block head %}
+{{ block.super }}
+{% load static %}
+<link rel="stylesheet" href="{% static 'SimpleInventory/css/orderboard.css' %}">
+{% endblock %}
+
+{% block js %}
+<div id="message-toast" class="mdl-js-snackbar mdl-snackbar">
+  <div class="mdl-snackbar__text"></div>
+  <button class="mdl-snackbar__action" type="button"></button>
+</div>
+{{ block.super }}
+{% load static %}
+<script src="{% static 'SimpleInventory/js/edit-board.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div id="formCard" class="form-card mdl-card mdl-cell--8-col mdl-cell--2-offset mdl-shadow--2dp">
```

Comment:
While the MDL grid has 12 columns on desktop, it has 8 on tablet-sizes and 4 on mobile.  Since the mdl-cell--2-offset adds padding to the left, this pushes the card off the screen on smaller devices.  I did not realize this was the case either, so this issue does exist on the pages I wrote elsewhere (which I can fix on those pages soon).

You can easily fix this by changing the `mdl-cell--2-offset` to `mdl-cell--2-offset-desktop` and adding the `mdl-grid` class to the formCard div.  Be sure to test in ~Firefox's~ your favorite web browser's Responsive Design Mode.  I'll do the same on the pages that introduced the bug.